src/recipes/heartbeat-by-user-first-impression/index.js

```diff
@@ -48,7 +48,7 @@ let { hasAny } = require("../../jetpack/array");
 */
 
 const NAME="heartbeat by user v1";
-const VERSION=15;
+const VERSION=16;
```

Comment:
This is a code refactoring to better handle l10n, but it shouldn't affect the outcome of the recipe. Should this version be increased?